mutant/modules/delivery.py

```diff
@@ -0,0 +1,190 @@
+""" This class modifies existing files to fit the Clinical Genomics infrastructure.
+    Specifically it acts on the sarscov2 pipeline.
+
+    By: Isak Sylvin & Tanja Normark
+
+"""
+
+import glob
+import os
+
+from mutant.modules.generic_parser import get_sarscov2_config
+
+
+class DeliverySC2:
+    def __init__(
+        self,
+        caseinfo: str,
+        indir: str,
+        nanopore: bool,
+        timestamp: str,
+        barcode_to_sampleid: dict,
+    ):
+        self.casefile = caseinfo
+        caseinfo = get_sarscov2_config(caseinfo)
+        self.caseinfo = caseinfo
+        self.case = caseinfo[0]["case_ID"]
+        self.ticket = caseinfo[0]["Customer_ID_project"]
+        self.indir = indir
+        self.nanopore = nanopore
+        self.timestamp = timestamp
+        self.barcode_to_sampleid = barcode_to_sampleid
+
+    def map_sampleid_to_barcode(self) -> dict:
+        sampleid_to_barcode = {}
+        for barcode in self.barcode_to_sampleid:
+            sampleid = self.barcode_to_sampleid[barcode]
+            sampleid_to_barcode[sampleid] = barcode
+        return sampleid_to_barcode
+
+    def translate_basename_to_nanopore_format(self, sampleid_to_barcode: dict) -> dict:
+        illunmina_to_nanopore_base = {}
```

Comment:
Maybe `illumina_to_nanopore_base` instead?

Reply:
should I name the function `translate_ullumina_to_nanopore_base`?

Reply:
Just a typo in `illunmina_to_nanopore_base = {} `

Reply:
aaah! now I see it :-)